db/seeds.rb

```diff
@@ -5,3 +5,107 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+# Users
+users = [
+    {
+        email: 'guerrero@email.com',
+        password: 'password'
+    },
+    {
+        email: 'fernando@email.com',
+        password: 'password'
+    },
+    {
+        email: 'lex@email.com',
+        password: 'password'
+    },
+    {
+        email: 'allen@email.com',
+        password: 'password'
+    }
+]
+
+users.each do |attributes|
+    User.create attributes
+end
+
+
+# Cards
+response = RestClient.get 'https://api.pokemontcg.io/v2/cards?q=set.id:base1'
+json = JSON.parse response
+puts json
+if !json.nil?
+  json["data"].map do |card|
+    if card["supertype"] == "Pokémon"
+      pokemon_type = card["types"][0]
+    else
+      pokemon_type = card["supertype"]
+    end
+    if card.key?("tcgplayer") == false
+      price = 0
+    else
+      if card["tcgplayer"]["prices"].key?("holofoil")
+        price = card["tcgplayer"]["prices"]["holofoil"]["high"]
+      elsif card["tcgplayer"]["prices"].key?("reverseHolofoil")
+        price = card["tcgplayer"]["prices"]["reverseHolofoil"]["high"]
+      elsif card["tcgplayer"]["prices"].key?("1stEditionHolofoil")
+        price = card["tcgplayer"]["prices"]["1stEditionHolofoil"]["high"]
+      elsif card["tcgplayer"]["prices"].key?("normal")
+        price = card["tcgplayer"]["prices"]["normal"]["high"]
+      else
+        price = 0
+      end
```

Comment:
```suggestion
      price_types = ["holofoil", "reverseHolofoil", "1stEditionHolofoil", "normal"]
      price = 0
      price_types.each do |type| 
        if card["tcgplayer"]["prices"].key?(type)
          price = card["tcgplayer"]["prices"][type]["high"]
        end
      end  
```

Reply:
you can store the keys that you care about in an array and iterate over the array to determine the price. Again, defaulting the value first to zero and then updating it if there's some particular conditional met. This way, if there's another key that you have to worry about, it's a matter of just adding it to the array instead of having to had another else if